src/paint.jsx

```diff
@@ -26,9 +26,10 @@ export default async function paint ($element, layout, component) {
     styling: { useSeparatorColumns }
   } = state;
 
+  const editmodeClass = component._inEditState ? 'edit-mode' : '';
```

Comment:
shouldn't we use `component.inEditState()` instead?

Reply:
nope, the property name is with underscore.
not using it did not work

Reply:
component.inEditState() would avoid the eslint warning and its not unusual for _ to denote a private variable, so the other version is probably less likely to break in the future.

Reply:
I was making the assumption based on another similar property updated by @Caele  in https://github.com/qlik-oss/QlikSense_Extension_2DimHeatmap/pull/41/files#diff-4f0be712c72e1f804a9ff53973e77a82R11